linter/whitelists/de.txt

```diff
@@ -43,6 +44,8 @@ Berechnungsarten
 Besuchererweiterung
 Bewertungssystem
 Bilddatenbank
+Bilder
+bilder
```

Comment:
Did it really complain about "Bilder"? Also adding `bilder` with a lower case `b` seems wrong.